### PR TITLE
getOutputAlias also returns the AliasInfo.

### PR DESCRIPTION
**Pull request by wujingyue**
This is separated out from #1203 to keep the PR small.

### CODE REVIEW

File: csrc/fusion.cpp [hunk 1]
```diff
@@ -773,9 +773,13 @@ void Fusion::aliasOutputToInput(Val* output, Val* input, const AliasType type) {
   }
 }
 
-Val* Fusion::getOutputAlias(Val* output) {
+Val* Fusion::getOutputAlias(Val* output, AliasInfo* info) {
   if (auto search = io_alias_.find(output); search != io_alias_.end()) {
-    return search->second.first;
+    const auto& [in_val, alias_info] = search->second;
+    if (info != nullptr) {
+      *info = alias_info;
```
Comment (jjsjann123) [inline csrc/fusion.cpp:780]: looks a bit ugly and we are copying here.
Maybe just return a pair of pointers?

**Review by jjsjann123 — APPROVED** [review]
nitpicking on style.

File: csrc/fusion.cpp (hunk 1 above)
Comment (wujingyue) [reply csrc/fusion.cpp]: Done.

**Comment by wujingyue** [comment]
!build

**Pull request merged**